app/mailers/application_mailer.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 class ApplicationMailer < ActionMailer::Base
-  default from: 'openhpi-support@hpi.uni-potsdam.de'
+  default from: 'CodeHarbor <openhpi-support@hpi.uni-potsdam.de>'
```

Comment:
Bonus: This looks nicer in email clients and makes the origin more clear. Is this overwritten in production?

Reply:
Nice addition, thanks. The value is overwritten in production but still didn't include the name. I changed it there as well. 🙏

Reply:
Nice, thx. :)